bodhi/server/push.py

```diff
@@ -50,8 +50,28 @@ def update_sig_status(update):
                 click.echo('Build %s still unsigned' % build.nvr)
 
 
+def check_if_updates_and_builds_set(ctx, param, value):
+    """
+    Print an error to stderr if the user has set both the --updates and --builds flags.
```

Comment:
It seems to me like it should be OK to mix these flags since they are OR'd together later - is there a problem you noticed when these flags are both used?

Reply:
When both flags will be provided, and both flags will be pointing to different updates, bodhi-push will not create any Composes.